src/wp-content/themes/twentytwenty/assets/css/editor-style-block.css

```
@@ -501,6 +501,17 @@
 	margin-top: 0.15em;
 }
 
+/* Block: Calendar --------------------------- */
+
+.editor-styles-wrapper .wp-calendar-table caption {
+	padding: 0.5em;
+	font-weight: 600;
+}
+:root .editor-styles-wrapper :where(.wp-block-calendar[style*="font-weight"]) caption, 
+:root .editor-styles-wrapper :where(.wp-block-calendar *[style*="font-weight"]) ~ caption {
+    font-weight: inherit;
+}
```

```suggestion
.editor-styles-wrapper caption {
	padding: 0.5em;
	font-weight: 600;
}

.editor-styles-wrapper :where(.wp-block-calendar[style*="font-weight"]) caption {
	font-weight: inherit;
}
```

The front-end styles add the padding and font weight to any `caption` element, so I would not restrict it to one block in the editor. However, adjusting for a custom font weight can be focused on the Calendar block (the standard Table block does not use the `caption` element).